demonstrations/tutorial_lcu_blockencoding.py

```diff
@@ -0,0 +1,272 @@
+r"""Linear combination of unitaries and block encodings
+=============================================================
+
+.. meta::
+    :property="og:description": Master the basics of LCUs and their applications
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_lcu_blockencoding.png
+
+.. related::
+
+    tutorial_intro_qsvt Intro to QSVT
+
+*Author: Juan Miguel Arrazola, Diego Guala, and Jay Soni — Posted: August, 2023.*
+
+If I (Juan Miguel) had to summarize quantum computing in one sentence, it would be this: information is
+encoded in quantum states and processed using `unitary operations <https://en.wikipedia.org/wiki/Unitary_operator>`_.
+The challenge of quantum algorithms is to design and build these unitaries to perform interesting and
+useful tasks. Now consider this. My colleague `Nathan Wiebe <https://scholar.google.ca/citations?user=DSgKHOQAAAAJ&hl=en>`_
+once told me that some of his early research was motivated by a simple
+question: Quantum computers can implement products of unitaries --- after all
+that's how we build circuits from a `universal gate set <https://en.wikipedia.org/wiki/Quantum_logic_gate#Universal_quantum_gates>`_.
+But what about **sums of unitaries**? 🤔
+
+In this tutorial you will learn the basics of one of the most versatile tools in quantum algorithms:
+linear combinations of unitaries; or LCUs for short. You will also understand how to
+use LCUs to create another powerful building block of quantum algorithms: block encodings.
+Among their many uses, they allow us to transform quantum states by non-unitary operators.
+Block encodings are useful in a variety of contexts, perhaps most famously in `qubitization <https://arxiv.org/abs/1610.06546)>`_ and the `quantum
+singular value transformation (QSVT) <https://pennylane.ai/qml/demos/tutorial_intro_qsvt>`_.
+
+|
+
+.. figure:: ../demonstrations/lcu_blockencoding/thumbnail_lcu_blockencoding.png
+    :align: center
+    :width: 50%
+    :target: javascript:void(0)
+
+|
+
+LCUs
+----
+The concept of an LCU is straightforward --- it’s already explained in the name: we
```

Comment:
"a" or just remove it:

```suggestion
The concept of LCU is straightforward --- it’s already explained in the name: we
```